assets/panel/debugger.properties

```diff
@@ -796,11 +796,11 @@ gotoLineModal.placeholder=Go to line…
 # to open the go to line modal
 gotoLineModal.title=Go to a line number in a file
 
-# LOCALIZATION NOTE(gotoLineModal.key2): The shortcut for opening the
+# LOCALIZATION NOTE(gotoLineModal.key3): The shortcut for opening the
 # go to line modal
-# Do not localize "CmdOrCtrl+;", or change the format of the string. These are
+# Do not localize "Ctrl+G", or change the format of the string. These are
 # key identifiers, not messages displayed to the user.
-gotoLineModal.key2=CmdOrCtrl+;
+gotoLineModal.key3=Ctrl+G
```

Comment:
hmm, cmd+g is another shortcut, should we limit it to cmd+g?

does case matter ctrl+g or ctrl+G?

Reply:
Any thoughts here @gregtatum